programs/dual-farming/src/calculator/pool_v1.rs

```diff
@@ -0,0 +1,140 @@
+use crate::calculator::*;
```

Comment:
Pls remove pool_v1, and rename pool_v2
Step finance has to update program, so they create v1, v2, but we deploy new program, then no need to use different versions

Reply:
Updated